src/rgw/rgw_acl.h

```diff
@@ -46,6 +46,9 @@ enum ACLGroupTypeEnum {
   ACL_GROUP_AUTHENTICATED_USERS = 2,
 };
 
+string get_acl_type_str (const ACLGranteeTypeEnum& type);
+ACLGranteeTypeEnum get_acl_type (const string& t);
+
```

Comment:
it looks like these are unused now that the encoding changed?

Reply:
Done. get_acl_type() is still used but as there is only one caller, I have moved it inline of that routine now.